doc/cookbook/splitting.md

```diff
@@ -66,3 +66,258 @@ $ pachctl put-file users master -c -f user_data.json --split json --target-file-
+
+$ pachctl get-file aaa master /looney_toons/0000000000000000
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.5.12
+-- Dumped by pg_dump version 9.5.12
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: looney_toons; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.looney_toons (
+    id integer NOT NULL,
+    name text NOT NULL,
+    saying text NOT NULL
+);
+
+
+ALTER TABLE public.looney_toons OWNER TO postgres;
+
+--
+-- Data for Name: looney_toons; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.looney_toons (id, name, saying) FROM stdin;
+0	wile E Coyote	...
+\.
+
+
+--
+-- PostgreSQL database dump complete
+--
+
+$ pachctl get-file aaa master /looney_toons
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.5.12
+-- Dumped by pg_dump version 9.5.12
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: looney_toons; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.looney_toons (
+    id integer NOT NULL,
+    name text NOT NULL,
+    saying text NOT NULL
+);
+
+
+ALTER TABLE public.looney_toons OWNER TO postgres;
+
+--
+-- Data for Name: looney_toons; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.looney_toons (id, name, saying) FROM stdin;
+\.
+
+
+--
+-- PostgreSQL database dump complete
+--
+
+$ pachctl get-file aaa master /looney_toons/**
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.5.12
+-- Dumped by pg_dump version 9.5.12
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: looney_toons; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.looney_toons (
+    id integer NOT NULL,
+    name text NOT NULL,
+    saying text NOT NULL
+);
+
+
+ALTER TABLE public.looney_toons OWNER TO postgres;
+
+--
+-- Data for Name: looney_toons; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.looney_toons (id, name, saying) FROM stdin;
+0	wile E Coyote	...
+1	road runner	meepmeep
+\.
+
+
+--
+-- PostgreSQL database dump complete
+--
```

Comment:
I'm lost trying to follow along with this whole example frankly. Please add more comments or break it the code block with text that describes the arguments and pachctl commend being made, what it's doing, and what they're seeing in the result. The alice/bob example was simple enough, but this needs more explanation.